lib/metrics/metrics.go

```diff
@@ -23,6 +23,7 @@ var (
 	HTTPReqSending    = stats.New("http_req_sending", stats.Trend, stats.Time)
 	HTTPReqWaiting    = stats.New("http_req_waiting", stats.Trend, stats.Time)
 	HTTPReqReceiving  = stats.New("http_req_receiving", stats.Trend, stats.Time)
+	HTTPReqTLSShaking = stats.New("http_req_tls_shaking", stats.Trend, stats.Time)
```

Comment:
Sorry for being a pita here, asking for a change again. I think you were correct in spelling out "handshaking" in the first round of commits, just saying "shaking" sounds strange to me, please change it to `http_req_tls_handshaking` (and `HTTPReqTLSHandshaking`).